src/App.jsx

```diff
@@ -12,7 +12,7 @@ function App() {
     <div className="main">
       <BrowserRouter>
         <Switch>
-          <Route path="/" exact>
```

Comment:
I'm "almost" sure this is an error, and was not tested (plus, it has nothing to do withthe PR's title)